include/openssl/srtp.h

```diff
@@ -28,16 +28,28 @@
 extern "C" {
 #endif
 
-# define SRTP_AES128_CM_SHA1_80 0x0001
-# define SRTP_AES128_CM_SHA1_32 0x0002
-# define SRTP_AES128_F8_SHA1_80 0x0003
-# define SRTP_AES128_F8_SHA1_32 0x0004
-# define SRTP_NULL_SHA1_80      0x0005
-# define SRTP_NULL_SHA1_32      0x0006
+# define SRTP_AES128_CM_SHA1_80                     0x0001
+# define SRTP_AES128_CM_SHA1_32                     0x0002
+# define SRTP_AES128_F8_SHA1_80                     0x0003
+# define SRTP_AES128_F8_SHA1_32                     0x0004
+# define SRTP_NULL_SHA1_80                          0x0005
+# define SRTP_NULL_SHA1_32                          0x0006
 
 /* AEAD SRTP protection profiles from RFC 7714 */
-# define SRTP_AEAD_AES_128_GCM  0x0007
-# define SRTP_AEAD_AES_256_GCM  0x0008
+# define SRTP_AEAD_AES_128_GCM                      0x0007
+# define SRTP_AEAD_AES_256_GCM                      0x0008
+
+/* DOUBLE AEAD SRTP protection profiles from RFC 8723 */
+# define DOUBLE_AEAD_AES_128_GCM_AEAD_AES_128_GCM   0x0009
+# define DOUBLE_AEAD_AES_256_GCM_AEAD_AES_256_GCM   0x000A
```

Comment:
There is some inconsistency here. The RFC 7714 profiles are identified in the RFC without the leading `SRTP_`, but the RFC8269 onces do have that. These new "double" protection profiles don't have the leading `SRTP_` which is kind of problematic, in that we are claiming a new namespace by not prefixing with SRTP. Should they be called `SRTP_DOUBLE_*` instead?

Reply:
That's a reasonable opinion, should I close this pull request and make another pull request with changes? Are there something clever way to make changes without another pull request?

Reply:
You can use
````
> git commit --fixup commit_hash
````

Get the commit_hash of any commit using
````
>git log
````

Reply:
So you can have multiple comitts for different things and then apply fixups to any commit when being reviewed.

Reply:
> 

Thanks, I'll give it a try